docs/source/tutorial/understanding_evaluation.rst

```diff
@@ -34,34 +34,77 @@ number.
 
 Rank-Based Metrics
 ~~~~~~~~~~~~~~~~~~
-Mean Rank (MR)
-**************
-The Mean Rank (MR)
+Given the set of individual rank scores $\mathcal{I}$, the following scores are commonly used as aggregation.
 
-It lies on the range $[0.0, \infty)$ where closer to 0 is better.
+Hits @ K
+********
+The hits @ k describes the fraction of true entities that appear in the first $k$ entities of the sorted rank list.
+It is given as:
+
+.. math::
+
+    \text{hits @ k} = \frac{|\{r \in \mathcal{I} \mid r \leq k\}|}{|\mathcal{I}|}
+
+For example, if Google shows 20 results on the first page, then the percentage of results that are relevant is the
+hits @ 20. The hits @ k, regardless of $k$, lies on the $[0.0, 1.0]$ where closer to 1 is better.
+
+.. warning::
+
+    This metric does not differentiate between cases when the rank is larger than $k$.
+    This means that a miss with rank $k+1$ and $k+d$ where $d \gg 1$ have the same
+    effect on the final score. Therefore, it is less suitable for the comparison of different
+    models.
+
+Mean Rank
+*********
+The mean rank (MR) computes the arithmetic mean over all individual ranks. It is given as:
+
+.. math::
+
+    \text{MR} =\frac{1}{|\mathcal{I}|} \sum \limits_{r \in \mathcal{I}} r
+
+It has the advantage over hits @ k that it is sensitive to any model performance changes, not only what occurs
+under a certain cutoff and therefore reflects average performance. It the mean rank lies on the
+range $[0.0, \infty)$ where closer to 0 is better.
```

Comment:
`[1, inf]` for 1-based rank

Reply:
we need a paragraph explaining 1 and 0-based ranking 😁

Reply:
Which one do we implement?

Reply:
We always use 1-based.

Reply:
Fixed in a014c08d. Thanks

Reply:
1-based. Only the AMRI uses 0-based, but only internally.